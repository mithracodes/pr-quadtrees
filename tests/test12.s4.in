144.952 -37.81 144.978 -37.79
144.953 -37.8095 144.9628 -37.7916
144.9628 -37.8095 144.97603 -37.7916
