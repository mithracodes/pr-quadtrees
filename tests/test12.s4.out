144.952 -37.81 144.978 -37.79
--> footpath_id: 16908 || address: Franklin Street between Elizabeth Street and Queen Street || clue_sa: Melbourne, CBD || asset_type: Road Footway || deltaz: 3.02 || distance: 58.88 || grade1in: 19.5 || mcc_id: 1387657 || mccid_int: 21634 || rlmax: 23.95 || rlmin: 20.93 || segside: South || statusid: 2 || streetid: 644 || street_group: 18332 || start_lat: -37.808944 || start_lon: 144.958117 || end_lat: -37.808772 || end_lon: 144.958836 || 
--> footpath_id: 19082 || address:  || clue_sa: West Melbourne, Residential || asset_type: Road Footway || deltaz: 0.42 || distance: 4.43 || grade1in: 10.5 || mcc_id: 1389319 || mccid_int: 0 || rlmax: 27.55 || rlmin: 27.13 || segside:  || statusid: 0 || streetid: 0 || street_group: 19082 || start_lat: -37.807053 || start_lon: 144.955545 || end_lat: -37.806493 || end_lon: 144.955644 || 
--> footpath_id: 19458 || address: Queensberry Street between Capel Street and Howard Street || clue_sa: North Melbourne || asset_type: Road Footway || deltaz: 1.44 || distance: 94.82 || grade1in: 65.8 || mcc_id: 1385878 || mccid_int: 20950 || rlmax: 35.75 || rlmin: 34.31 || segside: North || statusid: 2 || streetid: 1008 || street_group: 20939 || start_lat: -37.803461 || start_lon: 144.954243 || end_lat: -37.803556 || end_lon: 144.955388 || 
--> footpath_id: 19783 || address: Gatehouse Street between Morrah Street and Bayles Street || clue_sa: Parkville || asset_type: Road Footway || deltaz: 5.13 || distance: 139.97 || grade1in: 27.3 || mcc_id: 1388014 || mccid_int: 22342 || rlmax: 34.67 || rlmin: 29.54 || segside: East || statusid: 2 || streetid: 649 || street_group: 20509 || start_lat: -37.794582 || start_lon: 144.953825 || end_lat: -37.793647 || end_lon: 144.955069 || 
--> footpath_id: 20871 || address:  || clue_sa: Parkville || asset_type: Road Footway || deltaz: 1.98 || distance: 122.36 || grade1in: 61.8 || mcc_id: 1388567 || mccid_int: 0 || rlmax: 36.88 || rlmin: 34.90 || segside:  || statusid: 0 || streetid: 0 || street_group: 20873 || start_lat: -37.793948 || start_lon: 144.955843 || end_lat: -37.793165 || end_lon: 144.956543 || 
--> footpath_id: 22221 || address: Morrah Street between Fitzgibbon Street and Wimble Street || clue_sa: Parkville || asset_type: Road Footway || deltaz: 1.21 || distance: 61.48 || grade1in: 50.8 || mcc_id: 1387994 || mccid_int: 22325 || rlmax: 34.71 || rlmin: 33.50 || segside: South || statusid: 2 || streetid: 912 || street_group: 22867 || start_lat: -37.795780 || start_lon: 144.956659 || end_lat: -37.795699 || end_lon: 144.955851 || 
--> footpath_id: 24301 || address: Lygon Street between Victoria Street and Queensberry Street || clue_sa: Carlton || asset_type: Road Footway || deltaz: 1.02 || distance: 103.38 || grade1in: 101.3 || mcc_id: 1384568 || mccid_int: 20522 || rlmax: 36.75 || rlmin: 35.73 || segside: West || statusid: 2 || streetid: 840 || street_group: 25348 || start_lat: -37.805019 || start_lon: 144.966058 || end_lat: -37.806061 || end_lon: 144.965842 || 
--> footpath_id: 24585 || address:  || clue_sa: Carlton || asset_type: Road Footway || deltaz: 1.23 || distance: 76.82 || grade1in: 62.5 || mcc_id: 1386586 || mccid_int: 0 || rlmax: 33.35 || rlmin: 32.12 || segside:  || statusid: 0 || streetid: 0 || street_group: 25992 || start_lat: -37.800834 || start_lon: 144.957761 || end_lat: -37.801712 || end_lon: 144.957762 || 
--> footpath_id: 25302 || address:  || clue_sa: North Melbourne || asset_type: Road Footway || deltaz: 0.42 || distance: 13.21 || grade1in: 31.4 || mcc_id: 1386662 || mccid_int: 0 || rlmax: 26.93 || rlmin: 26.51 || segside:  || statusid: 0 || streetid: 0 || street_group: 25649 || start_lat: -37.801131 || start_lon: 144.955045 || end_lat: -37.800835 || end_lon: 144.954675 || 
--> footpath_id: 27665 || address: Palmerston Street between Rathdowne Street and Drummond Street || clue_sa: Carlton || asset_type: Road Footway || deltaz: 3.21 || distance: 94.55 || grade1in: 29.5 || mcc_id: 1384273 || mccid_int: 20684 || rlmax: 35.49 || rlmin: 32.28 || segside: North || statusid: 2 || streetid: 955 || street_group: 28597 || start_lat: -37.796156 || start_lon: 144.970564 || end_lat: -37.796061 || end_lon: 144.969417 || 
--> footpath_id: 28150 || address: Carlton Street between Nicholson Street and Canning Street || clue_sa: Carlton || asset_type: Road Footway || deltaz: 1.78 || distance: 142.03 || grade1in: 79.8 || mcc_id: 1388941 || mccid_int: 20683 || rlmax: 35.68 || rlmin: 33.90 || segside: South || statusid: 2 || streetid: 486 || street_group: 30384 || start_lat: -37.801109 || start_lon: 144.972475 || end_lat: -37.801267 || end_lon: 144.974145 || 
--> footpath_id: 28934 || address: Lygon Street between Argyle Place South and Pelham Street || clue_sa: Carlton || asset_type: Road Footway || deltaz: 0.24 || distance: 41.89 || grade1in: 174.4 || mcc_id: 1384628 || mccid_int: 20526 || rlmax: 38.91 || rlmin: 38.67 || segside: West || statusid: 2 || streetid: 840 || street_group: 30722 || start_lat: -37.803389 || start_lon: 144.966303 || end_lat: -37.802939 || end_lon: 144.966440 || 
--> footpath_id: 29340 || address:  || clue_sa: Carlton || asset_type: Road Footway || deltaz: 2.76 || distance: 58.83 || grade1in: 21.3 || mcc_id: 1465441 || mccid_int: 0 || rlmax: 33.28 || rlmin: 30.52 || segside:  || statusid: 0 || streetid: 0 || street_group: 29716 || start_lat: -37.801770 || start_lon: 144.962415 || end_lat: -37.802251 || end_lon: 144.962979 || 
--> footpath_id: 29735 || address:  || clue_sa: Carlton || asset_type: Road Footway || deltaz: 0.21 || distance: 33.23 || grade1in: 158.1 || mcc_id: 1466291 || mccid_int: 0 || rlmax: 39.12 || rlmin: 38.91 || segside:  || statusid: 0 || streetid: 0 || street_group: 30722 || start_lat: -37.803029 || start_lon: 144.965868 || end_lat: -37.802820 || end_lon: 144.966401 || 
--> footpath_id: 29778 || address:  || clue_sa: Carlton || asset_type: Road Footway || deltaz: 2.74 || distance: 87.05 || grade1in: 31.8 || mcc_id: 1466008 || mccid_int: 0 || rlmax: 33.84 || rlmin: 31.10 || segside:  || statusid: 0 || streetid: 0 || street_group: 30392 || start_lat: -37.800388 || start_lon: 144.972604 || end_lat: -37.800040 || end_lon: 144.973535 || 
--> footpath_id: 29996 || address:  || clue_sa: Carlton || asset_type: Road Footway || deltaz: 0.46 || distance: 54.51 || grade1in: 118.5 || mcc_id: 1388910 || mccid_int: 0 || rlmax: 24.91 || rlmin: 24.45 || segside:  || statusid: 0 || streetid: 0 || street_group: 29996 || start_lat: -37.793272 || start_lon: 144.975507 || end_lat: -37.794367 || end_lon: 144.975315 || 
--> footpath_id: 30009 || address: Canning Street between Palmerston Street and Pitt Street || clue_sa: Carlton || asset_type: Road Footway || deltaz: 0.62 || distance: 36.10 || grade1in: 58.2 || mcc_id: 1384191 || mccid_int: 20585 || rlmax: 28.08 || rlmin: 27.46 || segside: West || statusid: 2 || streetid: 479 || street_group: 30329 || start_lat: -37.796413 || start_lon: 144.972943 || end_lat: -37.795967 || end_lon: 144.973059 || 
--> footpath_id: 30062 || address:  || clue_sa: Carlton || asset_type: Road Footway || deltaz: 0.21 || distance: 43.47 || grade1in: 206.9 || mcc_id: 1384371 || mccid_int: 0 || rlmax: 44.71 || rlmin: 44.50 || segside:  || statusid: 0 || streetid: 0 || street_group: 30062 || start_lat: -37.801122 || start_lon: 144.969124 || end_lat: -37.801057 || end_lon: 144.968553 || 
--> footpath_id: 30352 || address: Drummond Street between Faraday Street and Elgin Street || clue_sa: Carlton || asset_type: Road Footway || deltaz: 2.18 || distance: 82.62 || grade1in: 37.9 || mcc_id: 1384188 || mccid_int: 20550 || rlmax: 38.67 || rlmin: 36.49 || segside: West || statusid: 2 || streetid: 583 || street_group: 30352 || start_lat: -37.797632 || start_lon: 144.968834 || end_lat: -37.798469 || end_lon: 144.968648 || 
--> footpath_id: 30703 || address: David Street between Elgin Street and Palmerston Street || clue_sa: Carlton || asset_type: Road Footway || deltaz: 0.60 || distance: 72.25 || grade1in: 120.4 || mcc_id: 1384348 || mccid_int: 20655 || rlmax: 30.26 || rlmin: 29.66 || segside:  || statusid: 3 || streetid: 558 || street_group: 30785 || start_lat: -37.797308 || start_lon: 144.971892 || end_lat: -37.796590 || end_lon: 144.972025 || 
144.953 -37.8095 144.9628 -37.7916
--> footpath_id: 16908 || address: Franklin Street between Elizabeth Street and Queen Street || clue_sa: Melbourne, CBD || asset_type: Road Footway || deltaz: 3.02 || distance: 58.88 || grade1in: 19.5 || mcc_id: 1387657 || mccid_int: 21634 || rlmax: 23.95 || rlmin: 20.93 || segside: South || statusid: 2 || streetid: 644 || street_group: 18332 || start_lat: -37.808944 || start_lon: 144.958117 || end_lat: -37.808772 || end_lon: 144.958836 || 
--> footpath_id: 19082 || address:  || clue_sa: West Melbourne, Residential || asset_type: Road Footway || deltaz: 0.42 || distance: 4.43 || grade1in: 10.5 || mcc_id: 1389319 || mccid_int: 0 || rlmax: 27.55 || rlmin: 27.13 || segside:  || statusid: 0 || streetid: 0 || street_group: 19082 || start_lat: -37.807053 || start_lon: 144.955545 || end_lat: -37.806493 || end_lon: 144.955644 || 
--> footpath_id: 19458 || address: Queensberry Street between Capel Street and Howard Street || clue_sa: North Melbourne || asset_type: Road Footway || deltaz: 1.44 || distance: 94.82 || grade1in: 65.8 || mcc_id: 1385878 || mccid_int: 20950 || rlmax: 35.75 || rlmin: 34.31 || segside: North || statusid: 2 || streetid: 1008 || street_group: 20939 || start_lat: -37.803461 || start_lon: 144.954243 || end_lat: -37.803556 || end_lon: 144.955388 || 
--> footpath_id: 19783 || address: Gatehouse Street between Morrah Street and Bayles Street || clue_sa: Parkville || asset_type: Road Footway || deltaz: 5.13 || distance: 139.97 || grade1in: 27.3 || mcc_id: 1388014 || mccid_int: 22342 || rlmax: 34.67 || rlmin: 29.54 || segside: East || statusid: 2 || streetid: 649 || street_group: 20509 || start_lat: -37.794582 || start_lon: 144.953825 || end_lat: -37.793647 || end_lon: 144.955069 || 
--> footpath_id: 20871 || address:  || clue_sa: Parkville || asset_type: Road Footway || deltaz: 1.98 || distance: 122.36 || grade1in: 61.8 || mcc_id: 1388567 || mccid_int: 0 || rlmax: 36.88 || rlmin: 34.90 || segside:  || statusid: 0 || streetid: 0 || street_group: 20873 || start_lat: -37.793948 || start_lon: 144.955843 || end_lat: -37.793165 || end_lon: 144.956543 || 
--> footpath_id: 22221 || address: Morrah Street between Fitzgibbon Street and Wimble Street || clue_sa: Parkville || asset_type: Road Footway || deltaz: 1.21 || distance: 61.48 || grade1in: 50.8 || mcc_id: 1387994 || mccid_int: 22325 || rlmax: 34.71 || rlmin: 33.50 || segside: South || statusid: 2 || streetid: 912 || street_group: 22867 || start_lat: -37.795780 || start_lon: 144.956659 || end_lat: -37.795699 || end_lon: 144.955851 || 
--> footpath_id: 24585 || address:  || clue_sa: Carlton || asset_type: Road Footway || deltaz: 1.23 || distance: 76.82 || grade1in: 62.5 || mcc_id: 1386586 || mccid_int: 0 || rlmax: 33.35 || rlmin: 32.12 || segside:  || statusid: 0 || streetid: 0 || street_group: 25992 || start_lat: -37.800834 || start_lon: 144.957761 || end_lat: -37.801712 || end_lon: 144.957762 || 
--> footpath_id: 25302 || address:  || clue_sa: North Melbourne || asset_type: Road Footway || deltaz: 0.42 || distance: 13.21 || grade1in: 31.4 || mcc_id: 1386662 || mccid_int: 0 || rlmax: 26.93 || rlmin: 26.51 || segside:  || statusid: 0 || streetid: 0 || street_group: 25649 || start_lat: -37.801131 || start_lon: 144.955045 || end_lat: -37.800835 || end_lon: 144.954675 || 
--> footpath_id: 29340 || address:  || clue_sa: Carlton || asset_type: Road Footway || deltaz: 2.76 || distance: 58.83 || grade1in: 21.3 || mcc_id: 1465441 || mccid_int: 0 || rlmax: 33.28 || rlmin: 30.52 || segside:  || statusid: 0 || streetid: 0 || street_group: 29716 || start_lat: -37.801770 || start_lon: 144.962415 || end_lat: -37.802251 || end_lon: 144.962979 || 
144.9628 -37.8095 144.97603 -37.7916
--> footpath_id: 24301 || address: Lygon Street between Victoria Street and Queensberry Street || clue_sa: Carlton || asset_type: Road Footway || deltaz: 1.02 || distance: 103.38 || grade1in: 101.3 || mcc_id: 1384568 || mccid_int: 20522 || rlmax: 36.75 || rlmin: 35.73 || segside: West || statusid: 2 || streetid: 840 || street_group: 25348 || start_lat: -37.805019 || start_lon: 144.966058 || end_lat: -37.806061 || end_lon: 144.965842 || 
--> footpath_id: 27665 || address: Palmerston Street between Rathdowne Street and Drummond Street || clue_sa: Carlton || asset_type: Road Footway || deltaz: 3.21 || distance: 94.55 || grade1in: 29.5 || mcc_id: 1384273 || mccid_int: 20684 || rlmax: 35.49 || rlmin: 32.28 || segside: North || statusid: 2 || streetid: 955 || street_group: 28597 || start_lat: -37.796156 || start_lon: 144.970564 || end_lat: -37.796061 || end_lon: 144.969417 || 
--> footpath_id: 28150 || address: Carlton Street between Nicholson Street and Canning Street || clue_sa: Carlton || asset_type: Road Footway || deltaz: 1.78 || distance: 142.03 || grade1in: 79.8 || mcc_id: 1388941 || mccid_int: 20683 || rlmax: 35.68 || rlmin: 33.90 || segside: South || statusid: 2 || streetid: 486 || street_group: 30384 || start_lat: -37.801109 || start_lon: 144.972475 || end_lat: -37.801267 || end_lon: 144.974145 || 
--> footpath_id: 28934 || address: Lygon Street between Argyle Place South and Pelham Street || clue_sa: Carlton || asset_type: Road Footway || deltaz: 0.24 || distance: 41.89 || grade1in: 174.4 || mcc_id: 1384628 || mccid_int: 20526 || rlmax: 38.91 || rlmin: 38.67 || segside: West || statusid: 2 || streetid: 840 || street_group: 30722 || start_lat: -37.803389 || start_lon: 144.966303 || end_lat: -37.802939 || end_lon: 144.966440 || 
--> footpath_id: 29340 || address:  || clue_sa: Carlton || asset_type: Road Footway || deltaz: 2.76 || distance: 58.83 || grade1in: 21.3 || mcc_id: 1465441 || mccid_int: 0 || rlmax: 33.28 || rlmin: 30.52 || segside:  || statusid: 0 || streetid: 0 || street_group: 29716 || start_lat: -37.801770 || start_lon: 144.962415 || end_lat: -37.802251 || end_lon: 144.962979 || 
--> footpath_id: 29735 || address:  || clue_sa: Carlton || asset_type: Road Footway || deltaz: 0.21 || distance: 33.23 || grade1in: 158.1 || mcc_id: 1466291 || mccid_int: 0 || rlmax: 39.12 || rlmin: 38.91 || segside:  || statusid: 0 || streetid: 0 || street_group: 30722 || start_lat: -37.803029 || start_lon: 144.965868 || end_lat: -37.802820 || end_lon: 144.966401 || 
--> footpath_id: 29778 || address:  || clue_sa: Carlton || asset_type: Road Footway || deltaz: 2.74 || distance: 87.05 || grade1in: 31.8 || mcc_id: 1466008 || mccid_int: 0 || rlmax: 33.84 || rlmin: 31.10 || segside:  || statusid: 0 || streetid: 0 || street_group: 30392 || start_lat: -37.800388 || start_lon: 144.972604 || end_lat: -37.800040 || end_lon: 144.973535 || 
--> footpath_id: 29996 || address:  || clue_sa: Carlton || asset_type: Road Footway || deltaz: 0.46 || distance: 54.51 || grade1in: 118.5 || mcc_id: 1388910 || mccid_int: 0 || rlmax: 24.91 || rlmin: 24.45 || segside:  || statusid: 0 || streetid: 0 || street_group: 29996 || start_lat: -37.793272 || start_lon: 144.975507 || end_lat: -37.794367 || end_lon: 144.975315 || 
--> footpath_id: 30009 || address: Canning Street between Palmerston Street and Pitt Street || clue_sa: Carlton || asset_type: Road Footway || deltaz: 0.62 || distance: 36.10 || grade1in: 58.2 || mcc_id: 1384191 || mccid_int: 20585 || rlmax: 28.08 || rlmin: 27.46 || segside: West || statusid: 2 || streetid: 479 || street_group: 30329 || start_lat: -37.796413 || start_lon: 144.972943 || end_lat: -37.795967 || end_lon: 144.973059 || 
--> footpath_id: 30062 || address:  || clue_sa: Carlton || asset_type: Road Footway || deltaz: 0.21 || distance: 43.47 || grade1in: 206.9 || mcc_id: 1384371 || mccid_int: 0 || rlmax: 44.71 || rlmin: 44.50 || segside:  || statusid: 0 || streetid: 0 || street_group: 30062 || start_lat: -37.801122 || start_lon: 144.969124 || end_lat: -37.801057 || end_lon: 144.968553 || 
--> footpath_id: 30352 || address: Drummond Street between Faraday Street and Elgin Street || clue_sa: Carlton || asset_type: Road Footway || deltaz: 2.18 || distance: 82.62 || grade1in: 37.9 || mcc_id: 1384188 || mccid_int: 20550 || rlmax: 38.67 || rlmin: 36.49 || segside: West || statusid: 2 || streetid: 583 || street_group: 30352 || start_lat: -37.797632 || start_lon: 144.968834 || end_lat: -37.798469 || end_lon: 144.968648 || 
--> footpath_id: 30703 || address: David Street between Elgin Street and Palmerston Street || clue_sa: Carlton || asset_type: Road Footway || deltaz: 0.60 || distance: 72.25 || grade1in: 120.4 || mcc_id: 1384348 || mccid_int: 20655 || rlmax: 30.26 || rlmin: 29.66 || segside:  || statusid: 3 || streetid: 558 || street_group: 30785 || start_lat: -37.797308 || start_lon: 144.971892 || end_lat: -37.796590 || end_lon: 144.972025 || 
